app/signout/signout.tsx

```
@@ -26,15 +24,11 @@ export default function SignOut() {
+		<button
 			onClick={handleSubmit}
-			color="primary"
-			variant="flat"
-			isLoading={isLoading}
-			isDisabled={isLoading}
-			className={styles.button}
+			className="w-full py-2 mt-4 text-center bg-red-500 text-white rounded hover:bg-red-600 transition-colors"
 		>
 			{isLoading ? "ログアウト中..." : "ログアウト"}
-		</Button>
+		</button>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)